pkg/clients/iam/role.go

```diff
@@ -175,12 +175,13 @@ func IsRoleUpToDate(in v1beta1.RoleParameters, observed iamtypes.Role) (bool, st
 		return false, "", err
 	}
 
+	ignoreFields := []string{"AssumeRolePolicyDocument"}
```

Comment:
Please inline this again or define it as `const`

Reply:
Updated the code as you requested.